### ISSUE TITLE
Inline map not supported :(

### ISSUE THREAD

**Issue by nowox** [ingydotnet/yaml-pm#155]
With a YAML like: 

```
---
foo: {bar, baz}
```

I get the `YAML_PARSE_ERR_INLINE_MAP`, this is sad because it is part of the YAML specifications.

**Comment by perlpunk** [comment]
Yeah, I think this one might not be trivial to fix. But you can always use explicit null values:
```
---
foo: {bar: null, baz: null}
```
